source/routing/raptor_solution_reader.cpp

```diff
@@ -560,16 +571,17 @@ std::ostream& operator<<(std::ostream& os, const Journey& j) {
 
 Solutions read_solutions(const RAPTOR& raptor,
                          const bool clockwise,
+                         const DateTime& departure_datetime,
                          const RAPTOR::vec_stop_point_duration& deps,
                          const RAPTOR::vec_stop_point_duration& arrs,
                          const bool disruption_active,
                          const type::AccessibiliteParams& accessibilite_params)
 {
     if (clockwise) {
-        return read_solutions(raptor, raptor_reverse_visitor(), deps, arrs,
+        return read_solutions(raptor, raptor_reverse_visitor(), departure_datetime, deps, arrs,
                               disruption_active, accessibilite_params);
     } else {
-        return read_solutions(raptor, raptor_visitor(), deps, arrs,
+        return read_solutions(raptor, raptor_visitor(), departure_datetime, deps, arrs,
```

Comment:
You can't, `raptor_reverse_visitor()` and `raptor_visitor()` don't have the same type.